d.tsv

```diff
@@ -1,6 +1,6 @@
 eco	name	pgn
 D00	Amazon Attack	1. d4 d5 2. Qd3
-D00	Blackmar Gambit	1. d4 d5 2. e4
+D00	Blackmar-Diemer Gambit	1. d4 d5 2. e4
```

Comment:
apparently it shows that a previous commit has changed the Blackmar-Diemer Gambit -> Blackmar Gambit https://github.com/lichess-org/scalachess/issues/183 .
chesscom and chesstempo says its blackmar-diemer so dont know what's correct

Reply:
I don't think it makes much sense to have 1. d4 d5 2. e4 be Blackmar Gambit a 1. d4 d5 2. e4 dxe4 3. Nc3 be Blackmar-Diemer Gambit? It's literally the same gambit, there is not even a way to get the pawn back on move 3.

Maybe we should just delete this?

Reply:
i guess Blackmar-Diemer is good to make it consistent

Reply:
This change was also part of https://github.com/lichess-org/scalachess/pull/214 a while ago and then reverted again (0dfe19c6deac9f4d785cf4abf11abfdf03c57b79) after after https://github.com/lichess-org/scalachess/issues/183#issuecomment-585678321.

Reply:
Interesting. I think that's an inconsistency in Chess365? At least I could not find any other sources on calling this Blackmar Gambit. [Chesstempo](https://old.chesstempo.com/gamedb/opening/196), for example, says Blackmar-Diemer-Gambit for this position (which makes sense to me, because ply 3 is where the gambit is offered).

I assume that historically the initial idea was called Blackmar-Gambit, and then further variations were developed by Diemer? At least that's how it sounds on the [German Wikipedia](https://de.wikipedia.org/wiki/Blackmar-Diemer-Gambit) (but the opening there is again a little bit different, which makes this even more confusing).

In any case, I think it would be best to just choose one name and stick with it, rather than jump around with naming.

Reply:
this is what [stackexchange ](https://chess.stackexchange.com/questions/40061/what-is-the-difference-between-blackmar-gambit-and-blackmar-diemer-gambit-or-ar)has to offer 😃

Reply:
Ok, cool. Let's go with the "practical" approach, then.